src/components/FileSelector/styled/index.js

```diff
@@ -156,7 +190,18 @@ export const StyledBackdrop = attachThemeAttrs(styled.div)`
         background: transparent;
         height: 100%;
         width: 100%;
-        `}
+        
+        ${props.size === 'large' &&
```

Comment:
we need to discuss with @TahimiLeonBravo how multiline will look like when using sizes